src/stapi_fastapi/routers/root_router.py

```diff
@@ -104,49 +133,82 @@ def __init__(
             tags=["Orders"],
         )
 
-    def get_root(self, request: Request) -> RootResponse:
-        return RootResponse(
-            id="STAPI API",
-            conformsTo=self.conformances,
-            links=[
-                Link(
-                    href=str(request.url_for(f"{self.name}:root")),
-                    rel="self",
-                    type=TYPE_JSON,
-                ),
-                Link(
-                    href=str(request.url_for(f"{self.name}:conformance")),
-                    rel="conformance",
-                    type=TYPE_JSON,
-                ),
-                Link(
-                    href=str(request.url_for(f"{self.name}:list-products")),
-                    rel="products",
-                    type=TYPE_JSON,
-                ),
-                Link(
-                    href=str(request.url_for(f"{self.name}:list-orders")),
-                    rel="orders",
-                    type=TYPE_JSON,
-                ),
+        if ASYNC_OPPORTUNITIES in conformances:
+            self.add_api_route(
+                "/searches/opportunities",
+                self.get_opportunity_search_records,
+                methods=["GET"],
+                name=f"{self.name}:list-opportunity-search-records",
+                summary="List all Opportunity Search Records",
+                tags=["Opportunities"],
+            )
+
+            self.add_api_route(
+                "/searches/opportunities/{search_record_id}",
+                self.get_opportunity_search_record,
+                methods=["GET"],
+                name=f"{self.name}:get-opportunity-search-record",
+                summary="Get an Opportunity Search Record by ID",
+                tags=["Opportunities"],
+            )
+
+    def get_root(self: Self, request: Request) -> RootResponse:
+        links = [
+            Link(
+                href=str(request.url_for(f"{self.name}:root")),
+                rel="self",
+                type=TYPE_JSON,
+            ),
+            Link(
+                href=str(request.url_for(f"{self.name}:conformance")),
+                rel="conformance",
+                type=TYPE_JSON,
+            ),
+            Link(
+                href=str(request.url_for(f"{self.name}:list-products")),
+                rel="products",
+                type=TYPE_JSON,
+            ),
+            Link(
+                href=str(request.url_for(f"{self.name}:list-orders")),
+                rel="orders",
+                type=TYPE_JSON,
```

Comment:
this should be geo+json -- also check that type is correctly returned by the endpoint